include/head/core/tasks/presence_sensing_driver_task.hpp

```diff
@@ -81,6 +83,8 @@ class PresenceSensingDriverMessageHandler {
             can_client.send_can_message(
                 can::ids::NodeId::host,
                 can::messages::PushToolsDetectedNotification{
+                    // 0 message index to indicate async message
```

Comment:
why?

Reply:
When the firmware sends a message that wasn't the result of a request we need a way to indicate that, so message_index = 0 is how I decided to do that. the way I wrote the monorepo side of this, it will never send a request with a message index of 0